DistributionSystemApi/DistributionSystemApi/Services/RecipientService.cs

```diff
@@ -0,0 +1,176 @@
+﻿using DistributionSystemApi.Data.Entities;
+using DistributionSystemApi.Data;
+using Microsoft.EntityFrameworkCore;
+using DistributionSystemApi.Requests;
+using DistributionSystemApi.Responses;
+
+namespace DistributionSystemApi.Services
+{
+    public class RecipientService
+    {
+        private readonly ContentDistributionSystemContext _context;
+
+        public RecipientService(ContentDistributionSystemContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<List<RecipientResponse>> GetRecipients(int page, int pageSize, CancellationToken cancellationToken)
```

Comment:
it would be great to have a type which describes a page instead of using just objects collection. It would be convenient for you as a front end developer to write pagination logic